src/components/table/table.stories.js

```diff
@@ -0,0 +1,132 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import {
+  boolean, text, select
+} from '@storybook/addon-knobs';
+import { action } from '@storybook/addon-actions';
+import { State, Store } from '@sambego/storybook-state';
+import countriesList from '../../../demo/data/countries';
+import Button from '../button';
+import MultiActionButton from '../multi-action-button';
+import {
+  Table, TableCell, TableHeader, TableRow
+} from './table';
+import OptionsHelper from '../../utils/helpers/options-helper';
+import notes from './notes.md';
+
+const store = new Store({
+  sortOrder: 'asc',
+  sortedColumn: '',
+  currentPage: '1'
+});
+
+const handleChange = (e, tableOptions) => {
+  const { sortOrder, sortedColumn, currentPage } = tableOptions;
+
+  store.set({ sortOrder, sortedColumn, currentPage });
+  action('change')(e, tableOptions);
+};
+
+const buildRows = (pageSizeFromKnobs) => {
+  const pageSize = pageSizeFromKnobs;
+  const currentPage = store.get('currentPage');
+
+  const endIndex = pageSize * currentPage;
+  const startIndex = endIndex - pageSize;
+  const rowsCountries = countriesList.slice(startIndex, endIndex);
+
+  const rows = [
+    <TableRow key='header' as='header'>
+      <TableHeader
+        sortable name='name'
+        scope='col'
+      >
+        Country
+      </TableHeader>
+
+      <TableHeader scope='col'>Code</TableHeader>
+    </TableRow>
+  ];
+
+  if (store.get('sortOrder') === 'asc') {
```

Comment:
Can probably refactor this `if` statement. If the `reverse` method is the only difference, that's the only thing you need to do in the statement. Everything else just needs to be written once.